src/controllers/user.js

```diff
@@ -0,0 +1,35 @@
+
+/* eslint-disable import/prefer-default-export */
+import jwt from 'jsonwebtoken';
```

Comment:
Please make your controllers class-based.

Reply:
Alright, That would be done